src/index.tsx

```diff
@@ -4,6 +4,15 @@ import reportWebVitals from './reportWebVitals';
 import { BrowserRouter } from 'react-router-dom';
 import Wrapper from './Components/Wrapper/Wrapper';
 import './index.css';
+import TagManager from 'react-gtm-module';
+
+const gtmId = process.env.REACT_APP_GOOGLE_ANALYTICS_ID;
+
+if (gtmId) {
```

Comment:
@msrezaie Perhaps you could move this to a function in `analytics.ts` and call it here.